app/models/host.rb

```diff
@@ -500,6 +500,24 @@ def mac_addresses
     hardware.nil? ? [] : hardware.mac_addresses
   end
 
+  def fingerprint
```

Comment:
@fdupont-redhat is this specific to an esx host?  If so I think we have a method in VMwareWebService to get this let me look into it.

Reply:
Yeah we do this [here](https://github.com/ManageIQ/vmware_web_service/blob/master/lib/VMwareWebService/esx_thumb_print.rb) which you can access through the [`vdlVcConnection`](https://github.com/ManageIQ/vmware_web_service/blob/master/lib/VMwareWebService/MiqVimVdlMod.rb#L31-L53).

This returns the actual vddk connection but we could split it up to add an API method to return the thumb print in its own method.